tests/foreman_test.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+FOREMAN_HOST = 'localhost'
+FOREMAN_PORT = 3000
+
+
+def test_foreman_service(host):
+    foreman = host.service("foreman")
+    assert foreman.is_running
+    assert foreman.is_enabled
```

Comment:
I saw @ekohl commenting that we should do an assert per test and thus combine them, e.g. `assert foreman.running and foreman.is_enabled`. Which style are you in favor of?

Reply:
I guess "it depends" :D

Pytest's `assert` is smart when it comes to displaying details of the assertion. Like when you do `assert a == b` it will actually display the diff if there is any (and you don't need `assert_eql` like in other frameworks…)
No idea if it's still smart if we start chaining things with `a and b`, but we can try that.

OTOH, I don't think it brings a lot of value in this particular example and in other cases I'd rather write multiple tests that use a shared fixture:
https://github.com/evgeni/foreman-quadlet/commit/ba8b616f659d5131a2f0f2e58eebb32934ee13ad

Reply:
> OTOH, I don't think it brings a lot of value in this particular example and in other cases I'd rather write multiple tests that use a shared fixture:
> [evgeni@ba8b616](https://github.com/evgeni/foreman-quadlet/commit/ba8b616f659d5131a2f0f2e58eebb32934ee13ad)

This looks nice and clean.

Reply:
I've used this approach here and in #11, so enjoy? :)